spec/configuration_spec.lua

```diff
@@ -125,6 +125,53 @@ describe('Configuration object', function()
       assert.same(services, filter_services(services, { '42' }))
       assert.same({}, filter_services(services, { '21' }))
     end)
+
+    describe("with service filter", function()
+      local mockservices = {
+        {id="42", hosts={"test.foo.com", "test.bar.com"}},
+        {id="12", hosts={"staging.foo.com"}},
+        {id="21", hosts={"prod.foo.com"}},
+      }
+
+      it("validates default case", function()
```

Comment:
I think we should be more specific when possible.
`it does not discard any service when there is not regex` or similar. What do you think?